packages/app/src/routers/SelectionRouter/SelectProject.tsx

```diff
@@ -113,32 +113,33 @@ const SelectProject = ({
         >
           <div className={"title-section idp-content-margins"}>
             <ButtonGroup>{createIconButton}</ButtonGroup>
-            <div className={"inline-input-with-button"}>
-              <LabeledInput
-                label={"Search"}
-                placeholder={"Will search in name or number"}
-                displayStyle={"inline"}
-                value={searchValue}
-                onChange={(event) => {
-                  const {
-                    target: { value },
-                  } = event;
-                  setSearchValue(value);
-                }}
-                onKeyDown={(event) => {
-                  if (event.key === "Enter") {
-                    startSearch();
-                  }
-                  if (event.key === "Escape") {
-                    setSearchValue("");
-                    setSearchParam("");
-                  }
-                }}
-              />
-              <IconButton onClick={startSearch}>
-                <SvgSearch />
-              </IconButton>
-            </div>
+            <LabeledInput
```

Comment:
Label above and button below looks a little strange:
![image](https://user-images.githubusercontent.com/38542573/172180159-606adc41-6bd3-4cd3-ae81-bd6ffad0aa5a.png)

Reply:
Did you have previous node_modules in the folder ? (And are  you using yarn?)

We need to use the yarn resolutions for the iTwinUI package to show up correctly.

Reply:
Looking good now after cleaning node_modules